packages/teams-js/test/utils.ts

```diff
@@ -99,6 +99,7 @@ export class Utils {
       },
       closed: false,
     };
+    global.fetch = jest.fn(() => Promise.resolve({ status: 200, ok: true } as Response));
   }
```

Comment:
>} [](http://example.com/codeflow?start=2&length=1)

Are there new tests to validate the fallback behavior, the list retrieval behavior, etc.?

Reply:
Yup, added unit tests for using the fallback when there is no supplied json object from the fetch call and if the fetch call itself fails.